data/json/mutations/mutations.json

```diff
@@ -4746,6 +4969,35 @@
     "description": "Your tongue is forked, like that of a reptile.  This has no effect.",
     "category": [ "LIZARD", "RAPTOR" ]
   },
+  {
+    "type": "mutation",
+    "id": "LONG_TONGUE",
+    "name": { "str": "Lolling Tongue" },
+    "points": 0,
+    "visibility": 1,
+    "ugliness": 3,
+    "changes_to": [ "LONG_TONGUE2" ],
+    "description": "Your tongue has lengthened and often extends far beyond your mouth.  This has no effect.",
+    "category": [ "BATRACHIAN" ]
+  },
+  {
+    "type": "mutation",
+    "id": "LONG_TONGUE2",
+    "name": { "str": "Whipping Tongue" },
+    "visibility": 1,
+    "description": "You've developed an extremely long and mobile tongue.  You may use it to attack your enemies.",
```

Comment:
Perhaps a spell for a longer-ranged and sticky version of this?

Reply:
I wasn't able to figure out the C++ needed to make it sticky to pull a monster in to you.   So I haven't done the spell version.